katello-attach-subscription

```diff
@@ -1121,8 +1138,10 @@ def vdcupdate()
         keep_virt_only = sub['keep_virt_only']
       end
 
-      # if the system is found, stop cyclyng over yaml,
-      break
+      # if the system is found and sub_layer is "stop_parsing", stop cyclyng over yaml
```

Comment:
what's about "override", shouldn't an "override" entry also stop the parsing?

Reply:
I thinked override only as a reset of the subscriptions parsed before and not also for stop parsing

Reply:
ok